packages/common/src/Utility.ts

```diff
@@ -117,14 +122,14 @@ export class SelectionBag extends SelectionBase {
         return retVal;
     }
 
-    set(itemArray) {
+    set(itemArray: ISelectionItem[]) {
         this.clear();
         itemArray.forEach(function (item) {
             this.append(item);
         }, this);
     }
```

Comment:
Is this intention with this method that we can just pass in an array of:
```
someWidget._selection.set([
    {
        _id: "some_arbitrary_string_id",
        element: d3SelectionOfAnElementWithADataRowBoundToIt
    }
])
```
...to set the selection for that widget?

Reply:
Yes the logic didn't change, just the syntax checking (which makes it easier when adding to new Widgets).

Reply:
(I was using it incorrectly in Graph2 and changing this highlighted it straight away)